sql/src/main/resources/sql_features.tsv

```diff
@@ -162,10 +162,10 @@ F034	Extended REVOKE statement	3	REVOKE statement to revoke a privilege that the
 F041	Basic joined table			YES		
 F041	Basic joined table	1	Inner join (but not necessarily the INNER keyword)	YES		
 F041	Basic joined table	2	INNER keyword	YES		
-F041	Basic joined table	3	LEFT OUTER JOIN	NO		
-F041	Basic joined table	4	RIGHT OUTER JOIN	NO		
-F041	Basic joined table	5	Outer joins can be nested	NO		
-F041	Basic joined table	7	The inner table in a left or right outer join can also be used in an inner join	NO		
+F041	Basic joined table	3	LEFT OUTER JOIN	YES		
+F041	Basic joined table	4	RIGHT OUTER JOIN	YES		
+F041	Basic joined table	5	Outer joins can be nested	YES		

```

Comment:
``F041-05: Subfeature F041-1 extended so that a <table reference> within the <joined table> can itself be a <joined table>``

so yes :)